recipes/_efa_install.rb

```diff
@@ -15,39 +15,41 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
-efa_tarball = "#{node['cfncluster']['sources_dir']}/aws-efa-installer.tar.gz"
+if node['conditions']['efa_supported']
```

Comment:
What about removing the nested if and use a guard like:
```ruby
return unless node['conditions']['efa_supported']
```

Reply:
good idea. done!